build.assets/tooling/cmd/gh-trigger-workflow/args.go

```diff
@@ -51,9 +52,22 @@ func (m inputMap) Set(s string) error {
 	return nil
 }
 
+// key holds the bytes of a GutHub app key, stored as an array of bytes.
```

Comment:
```suggestion
// key holds the bytes of a GitHub app key
```

Technically it's a slice, not an array. But that parts obvious anyway, so let's just omit it.

Reply:
Added comments as to why we care that its in a slice rather than a string.

Reply:
But it still says `GutHub` not `GitHub`.

Reply:
it's ok. guthub.com works too :trollface: